### DISCUSSION TITLE
10 reasons (you never knew existed) about why you should attend Universe

### DISCUSSION THREAD

**Discussion by tuves** [community/community#66508] [Announcements]
By now, you’re probably familiar with the work-related benefits of attending our one-of-a-kind [Universe](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=10-things) event: interacting with peers, hands-on learning, productivity tips, and certifications.

### But there’s more: Here are **10 surprising reasons** why GitHub Universe IRL is the place to be.

10. **The food**. Seriously. And did we mention that it’s all you can eat? 🍔
![food](https://github.com/community/community/assets/49915749/f1b12122-f4a9-433f-b990-99a6decf0bad)

9.) **The activities**. Last year, we had an AI portrait studio, Hubot predicting your future, ping pong, djs, and [live-coded music performances](https://youtu.be/A5sLbf7kZRU?si=ZeUB8sMFoPpV0g4G). Who knows what this year will bring? 🎨
![Experience](https://github.com/community/community/assets/49915749/6d1336a3-7354-4cc2-ab3d-2ddec1d7be22)

8.) **Stickers**. We all know that GitHub wouldn’t be GitHub without stickers. So get ready for tables chock-full of nerdy and adorable stickers. 🥰
![Stickers](https://github.com/community/community/assets/49915749/f18b03a0-3d50-4709-ad45-40d919f2911c)

7.) **Dibs on new [Shop](http://thegithubshop.com) drops**. From exclusive Universe merch to that [coveted Octocat light](https://www.instagram.com/p/Ck9TqczPwtv/), be the first to browse and purchase our latest Shop merchandise. 👕
![shop](https://github.com/community/community/assets/49915749/a8ecc97e-0d44-406b-a9ee-60968622087d)

6.) **The beauty**. Did you check out the pictures from last year? Rainbow domes, life-size tentacles, the stunning Mona stage theater, the prism light experience. You can’t get more sensational. 🌈

5.) **Experts at your disposal**. Got a burning question? Our GitHub Ask booth is here for you + the demo stage, which was a favorite last year. 🤓
![Ask GitHub](https://github.com/community/community/assets/49915749/2141ec33-4e7f-4bcb-a2c5-3cf3b9aa2f01)

4.) **Comfortable weather**. San Francisco typically has mild weather in the fall – with moderate temps and refreshing rain and sunshine breaks. ☀️

3.) **Chance to be # 1**. One-up your colleagues by being the first to find out about all our latest product releases. 💪

2.) **New BFFs**. Enjoy some great networking opportunities, make new friends, and chat with our super-friendly GitHub team. 🫱🏼‍🫲🏾

1.) **The donuts**. Check out [this picture](https://twitter.com/jzmusings/status/1592601641290637313). Need we say more? 🍩

So what are you waiting for? [Buy your Universe ticket today](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=10-things). And don’t delay: Our Early Bird 20% off discount is still available!

**Comment by reeveng** [comment]
Why should they revert the recent changes to the GitHub homepage? Because it's the only way to ensure that the 'Donuts' category is back in full force at Universe! 🍩😄 

Also ensures that the devs that actually use the platform donut (🍩) go nuts!

**Comment by memochou1993** [comment]
Please bring back the old feed!

**Reply by xieyuheng** [reply]
Please bring back the old feed!!

**Comment by ghost** [comment]
so you're really just gonna ignore this thread:

https://github.com/orgs/community/discussions/66188

with over 1000 votes? its now in the top 5 all time threads. you're really just gonna move on @tuves and say nothing?

update. apparently they are refusing to fix this:

https://github.com/orgs/community/discussions/65343#discussioncomment-6983114

**Reply by Rifatmond** [reply]
Help me

**Reply by see7e** [reply]
3*e

**Reply by tuves** [reply]
@1268, in an effort to bring news (new ships, betas, upgrades, etc) into the community, we've made an effort to share certain [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) announcements here in Community Discussions. However, it doesn't mean that what we amplify here are things that I or my immediate team are directly working on. There are many Product and Engineering teams involved in any given release. I can assure you that the proper teams are looking into the feedback shared, even if they aren't answering every single post.

**Reply by ghost** [reply]
@tuves there's a big difference between answering "every single post", and answering none of them, which is the current case here. if ANY of the threads regarding the new feed have an official response, I would be interested, but I dont think thats the case.

**Reply by reeveng** [reply]
In only 4 days, the main request to revert the new feed has become the 3th most upvoted discussion, whilst the other discussions have been there for years, granted those probably also gained massive attraction at first and less later on, its just an overwhelming response without ANY response on the feedback, not even (until now/here) a "we're looking into it".

Yes, Github is bigger than one team and feedback loops can take a while and have to go through the appropriate hoops, no one is denying that, its just frustrating that it apparently takes that long.

As mentioned in other threads by other users, if you'd work with issues/states no/less frustration would arise, granted a potential amount of feedback wouldn't be given since people would assume you're looking into it already, but could be displaced with having a state purely for garnering feedback.

**Reply by fdwr** [reply]
@1268: I wager that on Nov 8th in San Francisco, there will be plenty of IRL feedback about the feed change (and IRL is harder to ignore).

**Reply by ghost** [reply]
in one week its now the number two discussion of ALL TIME, going back to at least 2017.

**Reply by infection00** [reply]
Have a cry ya sook

**Comment by xieyuheng** [comment]
Please bring back the old feed!

**Comment by memochou1993** [comment]
Now I can see who has starred my repos, but it can only display content from the previous week.

<img width="712" alt="image" src="https://github.com/community/community/assets/38420282/9aa05e06-f52e-4fb9-8d98-bd7c394d098a">

**Comment by infection00** [comment]
I believe the information I was looking for is available but being kept
from me
Which I think is an injustice being google has been sucking every piece of
data ive produced in the last ten years, I will be going fully open
scorce/google frds for my next Android with a hefty distance between gits
as I believe they are becoming data thirsty too, my ass hole ex girl has
been spilling my data since we split yet I'm being data farmed I should
have every right to see the info but they won't provide it
I do appreciate they contact though

**Comment by realitystevens** [comment]
Is there an opportunity to connect virtually?

**Reply by tuves** [reply]
@realitystevens Yes, there will be plenty of opportunities in Community Discussions. Stay tuned :)

**Reply by realitystevens** [reply]
This is great. With this, I'm hoping to participate in the Universe event.

**Comment by Kishlol** [comment]
kish@1984_[]()_

**Comment by novium2** [comment]
Closing this as it is almost time for Universe 2024! If you have any questions, start a new [discussion](https://github.com/orgs/community/discussions/new/choose)
